### ISSUE TITLE
VS 2022 17.6.2 IOS Simulator/Device not displaying correctly

### ISSUE THREAD

**Issue by techbud** [dotnet/maui#15487]
### Description

I'm not sure if I'm supposed to post this here or not. But after updating to 17.6.2 IOS and Android do not render the same. Android is rendered the same as I had it, but IOS is not the same and the buttons I have are pushed off-screen.

### Steps to Reproduce

1. Update to 17.6.2
2. Run an app on IOS Simulator/Device

### Link to public reproduction project repository

don't have one

### Version with bug

7.0.81

### Last version that worked well

7.0.81

### Affected platforms

iOS

### Affected platform versions

iOS 16.4

### Did you find any workaround?

Not as of yet.

**Comment by jadenrogers** [comment]
@techbud No sample provided, but does it involve ScrollViews? There are known issues with ScrollViews on ios.

**Comment by techbud** [comment]
> @techbud No sample provided, but does it involve ScrollViews? There are known issues with ScrollViews on ios. 
> Can you provide the XAML?

I will take a look when I get home. I have for sure a grid with two rows, top row *, Auto on the second and a collection view inside.

But will get code when I get home tonight.

**Comment by vredenbaugh-extron** [comment]
seeing the same issues. buttons that were working before are now pushed off the screen with the update. I am unable to click the bottom 1/3 portion of the screen with this update as well

**Comment by PureWeen** [comment]
Can you run

`dotnet workload list`

The latest version of MAUI is 7.0.86 which should have these issues resolved.

**Comment by vredenbaugh-extron** [comment]
> Can you run
> 
> `dotnet workload list`
> 
> The latest version of MAUI is 7.0.86 which should have these issues resolved.

The latest version does not fix these issues for me. I have 7.0.86

**Comment by techbud** [comment]
I updated the issue with my code, also I'm on 7.0.86 after reupdating to the latest version

**Comment by jadenrogers** [comment]
> I updated the issue with my code, also I'm on 7.0.86 after reupdating to the latest version

Is that the full Content? Anything in the RowDefinitions?

**Comment by techbud** [comment]
> > I updated the issue with my code, also I'm on 7.0.86 after reupdating to the latest version
> 
> Is that the full Content? Anything in the RowDefinitions?

Yes, not usre why it's not showing, shows in the brackets when I edit it but not when I save, but it's
  <RowDefinition Height="*"/>
 <RowDefinition Height="Auto"/>

**Comment by jadenrogers** [comment]
@techbud Yeh so this issue is logged somewhere, Try wrap the CollectionView in a DockLayout you will need to add the MAUI Community toolkit 

```
<mct:DockLayout Grid.Row="0">
    <CollectionView></CollectionView>
</mct:DockLayout>
```

**Comment by techbud** [comment]
> @techbud Yeh so this issue is logged somewhere, Try wrap the CollectionView in a DockLayout you will need to add the MAUI Community toolkit (As a workaround)
> 
> ```
> <mct:DockLayout Grid.Row="0">
>     <CollectionView></CollectionView>
> </mct:DockLayout>
> ```

So this workaround does seem to work. have so many workarounds
